README.md

```diff
@@ -1,5 +1,41 @@
 # jenkins-head
 This repository contains gatt-server and gatt-client software. Main goal is to visualize the Jenkins state :) 
 
-# The GATT-server is based on the project of Douglas Otwell
+##The GATT-server is based on the project of Douglas Otwell
 https://github.com/douglas6/cputemp
+
+# system overview
+```
+                                                       +-------------------+
+                                                       |                   |
++-------------+            +--------------+            | +--------------+  |
+|             |            |              |            | |              |  |
+|   Jenkins   |            |     GATT     |            | |     GATT     |  |
+|   Server    <------------+    Server    <-------------->    Client    |  |
+|             |    REST    |              |     BLE    | |              |  |
++-------------+    API     +--------------+            | +--------------+  |
+                                                       |      Jenkins      |
+                                                       |       Head        |
+                                                       |                   |
+                                                       +-------------------+
+```
+
+## GATT Server
+The GATT Server collects the state of all registered Jenkins jobs. After that the server joins all states to one global state. This global state is represented as an **RGB** value to an GATT Service.
+
+### Jenkins jobs
+The jobs that should be monitored are represented as an entry in a YAML file e.g.
+```yaml
+- jobName: "job1"
+  jobUrl: "http://..."
+  
+- jobName: "job2"
+  jobUrl: "http://..."
+```
+### GATT characteristics
+TBD
+
+## GATT Client
+The GATT Client is colouring the jenkins head regarding the state. 
+- red => There are errors
+- green => Every thing is OK
```

Comment:
If a person has a green head he/she is probably sick. So the head should be white if everything is OK.

To keep it as simple as possible the head should just read the **RGB** values from the GATT Server.
It should not know the different states.